docs/learn-legitimate-projects.md

```diff
@@ -1,242 +1,162 @@
-image search, if you want to be thorough.
+This is especially true for projects that are already running on another network, like on Ethereum or Binance Smart Chain, and have issued tokens there. Many projects do that either to raise funds and test their infrastructure or because they aim to build a "multi-chain" solution or both. But because those projects aren't currently built on Substrate, the existence of a clear and robust integration plan with Polkadot should be essential in your research to ensure that they will indeed build on Polkadot one day.
 
-Of course, a red flag is also if the team members aren't mentioned at all, and can be found only in
-their Telegram group under pseudonyms.
+## "Soft" metrics
+The items above are what you should look at first when evaluating a project and should carry most of the weight in your decision. The reason is that they are hard to fake or manipulate, provided, of course, that you are able to verify the information found. Hence we called them "hard" metrics.
 
-#### 3. Don't trust social media engagement
+But there are other things to look for that might point to a project's legitimacy but can be more easily manipulated, so they shouldn't affect your decision heavily. These are called "soft" metrics.
 
-Although having a vibrant community is a good indication of a legitimate project, many fake projects
-will use bots to populate their channels to show high member count and engagement. This is
-particularly common in Telegram and Discord, although according to Discord terms any bot users need
-to be idintified as such, but scammers don't really care about terms and conditions.
+#### 1. Site quality
 
-So, don't rely just on Telegram or Discord engagement and what people say in these channels. They
-could very easily be bot accounts.
+The quality of a project's site could sometimes provide insights into the legitimacy of a project. A poorly constructed site, with typos and grammatical errors, or poor styling, a site that's a template without any serious effort to improve or change it, a site that holds little information about the project, without links to their GitHub or other resources, and generally a site that doesn't feel professional, are indications that the team is not serious about this project.
 
-#### 4. Public provider emails
+But that doesn't mean that all well-designed sites are also solid projects. This is a soft metric, after all. Many projects that don't have any plans to build anything substantial still have excellent, or even beautiful-looking, sites. They put many resources into how they present themselves visually to mislead. So, an excellent site doesn't necessarily indicate a legitimate project, a poor site might indicate an illegitimate one, but the site quality alone usually isn't enough to reach a conclusion. None of these metrics are; you need to look into all of them to make an educated decision.
```

Comment:
Take out "But".

Reply:
Hehe. Our writing style clearly differs in that regard. I like to continue the train of thought from one paragraph to the other.